### DISCUSSION TITLE
A typebox api client :)

### DISCUSSION THREAD

**Discussion by flodlc** [sinclairzx81/typebox#474] [Show and tell]
I would like to share with you my work on a typesafe api client made around Typebox called typebox-client.

[https://github.com/flodlc/typebox-client](https://github.com/flodlc/typebox-client)

I use it in many monorepos projects and decided to create a library for easy reusability.
Thanks for your further suggestions and feedbacks !

**Comment by sinclairzx81** [comment]
@flodlc Hi, sorry for the delay in response, this look great!

Hey, happy to include `typebox-client` in the ecosystem section of the readme if you're interested. I imagine this client would be very useful for Fastify users! :) Let me know.

Great work!
S

**Reply by flodlc** [reply]
I would really appreciate it!
I will release a version very soon that improves the api and fix somes issues. 

Thanks for the support 🙏

**Reply by sinclairzx81** [reply]
@flodlc Cool, Have added :) Good work!
